doctr/utils/metrics.py

```diff
@@ -441,7 +312,7 @@ def summary(self) -> Tuple[Optional[float], Optional[float], Optional[float]]:
         precision = self.matches / self.num_preds if self.num_preds > 0 else None
 
         # mean IoU
-        mean_iou = self.tot_iou / self.num_preds if self.num_preds > 0 else None
+        mean_iou = round(self.tot_iou / self.num_preds, 2) if self.num_preds > 0 else None
```

Comment:
why rounding ?

Reply:
Because there may be minimal calculation deviations like 0.12999999......8 instead of 0.13 for example